src/test/scala/mesosphere/marathon/core/instance/InstanceFormatTest.scala

```diff
@@ -15,9 +15,9 @@ class InstanceFormatTest extends UnitTest {
       |{
       |  "instanceId": { "idString": "app.instance-1337" },
       |  "tasksMap": {},
-      |  "runSpecVersion": "2015-01-01",
+      |  "runSpecVersion": "2015-01-01T12:00:00Z",
```

Comment:
I am a little bit woried that if we had to change so manye values in tests we will break the behavior for our clients... or what am I missing?

Reply:
I agree. This should not change.

Reply:
In our api we have datetime type for this field, so it's impossible to have such data passed our validation layer.

Reply:
valid, point, thanks 👍